pandas/core/nanops.py

```diff
@@ -162,6 +162,9 @@ def f(
 def _bn_ok_dtype(dtype: DtypeObj, name: str) -> bool:
     # Bottleneck chokes on datetime64, PeriodDtype (or and EA)
     if not is_object_dtype(dtype) and not needs_i8_conversion(dtype):
+        # GH 42878
+        # Bottleneck uses naive summation leading to O(n) loss of precision
+        # unlike numpy which implements pairwise summation, which has O(log(n)) loss
```

Comment:
One more item: Is there a issue on the bottleneck Github about the precision issue that you can comment here?

Reply:
I believe pydata/bottleneck/issues/379 is the pertinent issue